src/parser/cvc/cvc_input.cpp

```diff
@@ -63,7 +63,7 @@ Command* CvcInput::parseCommand() {
 
 api::Term CvcInput::parseExpr()
 {
-  return api::Term(d_pCvcParser->parseExpr(d_pCvcParser));
+  return d_pCvcParser->parseExpr(d_pCvcParser);
```

Comment:
This is weird. Why do we pass `d_pCvcParser` here?

Reply:
I guess this is created for the input, https://github.com/CVC4/CVC4/blob/master/src/parser/cvc/cvc_input.h#L40, seems a bit backwards to me too.